closure/compiler/test/strict_dependency_checking/BUILD

```diff
@@ -245,6 +245,22 @@ genrule(
     cmd = "echo 'import { g } from \"/closure/compiler/test/strict_dependency_checking/es6_g.js\"; g();' >$@",
 )
 
+genrule(
+    name = "es6_g3_generate",
+    outs = ["es6_g3.js"],
+    cmd = "echo 'import { g } from \"./es6_g.js\"; g();' >$@",
+)
+
+closure_js_library(
+    name = "es6_g3_gen_lib",
+    srcs = ["es6_g.js", ":es6_g3_generate"],
+)
+
+closure_js_binary(
```

Comment:
I think you need to add a `file_test` that asserts the content of the resulting binary

Reply:
Done